src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationDownloadServiceImpl.java

```diff
@@ -44,14 +48,29 @@ public AnnotationResponseDto downloadAnnotationDoc(Integer annotationId, Integer
             throw new DartsApiException(INVALID_ANNOTATIONID_OR_ANNOTATION_DOCUMENTID);
         }
 
-        final ExternalObjectDirectoryEntity externalObjectDirectoryEntity = eodDir.get(0);
+        final ExternalObjectDirectoryEntity latestExternalObjectDirectoryEntity = eodDir.get(0);
+
+        var downloadableExternalObjectDirectories = DownloadableExternalObjectDirectories
+            .getFileBasedDownload(eodDir);
+
+        dataManagementFacade.getDataFromUnstructuredArmAndDetsBlobs(downloadableExternalObjectDirectories);
+        var downloadResponseMetaData = downloadableExternalObjectDirectories.getResponse();
 
-        final InputStreamResource blobStream = annotationDataManagement.download(externalObjectDirectoryEntity);
```

Comment:
Does this make `AnnotationDataManagement.download` redundant? Would the call out to the facade not be better in there?

Reply:
yes - it can go in there - I don't think this was here when i wrote my original changes